RadioIDs.py

```diff
@@ -46,12 +46,13 @@ def _scrape(sources: List[str]) -> List[str]:
         for source, info in data.items():
             namecache[str(source)] = info
             names.append(info)
+            log.info(f"Found names: {names}")
```

Comment:
This will print the full current list of names for every iteration, which might be a bit redundant

Reply:
hnnnnggggg